lib/rucio/core/transfer.py

```diff
@@ -556,13 +556,13 @@ def update_transfer_state(
             return nb_updated
     except UnsupportedOperation as error:
         logger(logging.WARNING, "Request %s doesn't exist - Error: %s" % (request_id, str(error).replace('\n', '')))
-        return 0
     except Exception:
         logger(logging.CRITICAL, "Exception", exc_info=True)
+    return 0
```

Comment:
This is also a functional change, related to the previous one - please rebase this one in a separate commit as well (ideally the same commit as the other change in this function)